src/plugin/code.js

```diff
@@ -27,7 +27,8 @@ const renderDifferenceRectangle = (differences, modifiedFrames) => {
 
       const differenceRectangle = figma.createRectangle();
 
-      differenceRectangle.resize(width, height);
+      differenceRectangle.name = "Difference Rectangle Node";
+      differenceRectangle.resize(width || 1, height || 1);
```

Comment:
P2: 1이라는 숫자도 상수처리해도 좋지 않을까요?

```suggestion
      differenceRectangle.resize(width || CONSTANT.MIN_WIDTH, height || CONSTANT.MIN_HEIGHT);
```